codemon/CodemonMeta.py

```diff
@@ -36,9 +36,47 @@ def default_cpp(self):
   def default_py(self):
     now = datetime.now()
     template="""
-# author: @ankingcodes
+# author: @Siddhant-K-code

```

Comment:
@Siddhant-K-code - Don't mind it but could you replace author to the previous ? Since it's a default template generated by codemon, I want it to be that way. But people do want to use their own name, so we have a command `codemon reg` which will register specific details and store in a folder `.codemon` at the root of your directory system. I wanted to add a feature which will get the username of the user from that folder if the user has registered else just use "@ankingcodes".

However, you can simply replace the name now for this PR and get this merged. You can create an issue on the feature I have described above and make a PR for it (bonus points 💥)

Reply:
Author Name Updated to `@ankingcodes`